_toc.yml

```diff
@@ -61,6 +62,10 @@ entries:
           - file: docs/platform/howto/monitoring-services
           - file: docs/platform/howto/integrations/prometheus-metrics
           - file: docs/platform/howto/integrations/datadog-increase-metrics-limit
+      - file: docs/platform/howto/list-billing
```

Comment:
We have this section for billing as well on the concepts side. So maybe we need to adopt just one certain term for that.

      - file: docs/platform/concepts/list-billing
        title: Billing